test/SdkUnitTests.js

```diff
@@ -1295,4 +1296,344 @@ describe('SDK Unit Tests:', function (done) {
         }
       });
   });
+
+  it("Get envelope by id", function (done) {
+    var envelopesApi = new docusign.EnvelopesApi(apiClient);
+
+    envelopesApi.getEnvelope(accountId, envelopeId)
+      .then(function (envelope) {
+        assert.notStrictEqual(envelope, undefined);
+        done();
+      })
+      .catch(function (error) {
+        if (error) {
+          return done(error);
+        }
+      });
+  });
+  
+  it('Get envelope recipients', function(done) {
+    var envelopesApi = new docusign.EnvelopesApi(apiClient);
+
+    envelopesApi.listRecipients(accountId, envelopeId)
+      .then(function (recipients) {
+        assert.notStrictEqual(recipients, undefined);
+        done();
+      })
+      .catch(function (error) {
+        if (error) {
+          return done(error);
+        }
+      });      
+  })
+
+  it('Create sender view', function(done) {
+    var fileBytes = null;
+    try {
+      var fs = require('fs');
```

Comment:
Why are we reimported the **fs** module 17 times in the tests?

Line 15 `var fs = require('fs');`